app/models/manageiq/providers/ansible_tower/automation_manager/credential.rb

```diff
@@ -1,3 +1,11 @@
 class ManageIQ::Providers::AnsibleTower::AutomationManager::Credential < ManageIQ::Providers::ExternalAutomationManager::Authentication
+  # Authentication is associated with EMS through resource_id/resource_type
+  # Alias is to make the AutomationManager code more uniformly as those
+  # CUD operations in the TowerApi concern
+
+  alias_attribute :manager_id, :resource_id
```

Comment:
it's being used [here](https://github.com/jameswnl/manageiq/blob/6f5a16e5b52061b82a8abc862982b93348930a2d/app/models/manageiq/providers/ansible_tower/shared/automation_manager/tower_api.rb#L11-L11)